.github/workflows/TrustKotlinGradlePluginPatch.main.kts

```diff
@@ -10,6 +10,7 @@ import kotlin.io.path.listDirectoryEntries
 import org.w3c.dom.Document
 import org.w3c.dom.Element
 
+//todo delete if not useful
```

Comment:
I think we should delete this file. Could you please also delete the hacks on the release workflow?